dev-requirements.txt

```diff
@@ -1,22 +1,18 @@
 mock==2.0.0
-coverage==4.5.2
-twine==1.11.0
+coverage~=4.5
+tox==2.9.1
 wheel==0.30.0
 tornado==5.0.2
 PySocks==1.6.8
 pkginfo==1.4.2
-pytest-random-order==1.0.4;python_version>="3.5"
```

Comment:
This is useful to see tests that depend on each other, eg. it allowed me to fix `test_create_connection_timeout` which depended on other tests. https://github.com/urllib3/urllib3/pull/1554/commits/cac3028dffbb112250909f3e83326efb170030b5

I think we can leave this like this